core/src/main/java/bio/terra/pearl/core/service/export/EnrolleeExportService.java

```diff
@@ -90,16 +94,20 @@ public Map<String, String> generateExportMap(EnrolleeExportData exportData,
         return valueMap;
     }
 
+    /**
+     * gets information about the modules, which will determine the columns needed for the export
+     * e.g. the columns needed to represent the survey questions.
+     */
 
-    public List<ModuleExportInfo> generateModuleInfos(ExportOptions exportOptions, UUID portalId, UUID studyEnvironmentId) throws Exception {
+    public List<ModuleExportInfo> generateModuleInfos(ExportOptions exportOptions, UUID studyEnvironmentId) throws Exception {
         List<ModuleExportInfo> moduleInfo = new ArrayList<>();
         moduleInfo.add(new EnrolleeFormatter().getModuleExportInfo(exportOptions));
         moduleInfo.add(new ProfileFormatter().getModuleExportInfo(exportOptions));
-        moduleInfo.addAll(generateSurveyModules(exportOptions, portalId, studyEnvironmentId));
+        moduleInfo.addAll(generateSurveyModules(exportOptions, studyEnvironmentId));
         return moduleInfo;
     }
 
-    protected List<ModuleExportInfo> generateSurveyModules(ExportOptions exportOptions, UUID portalId, UUID studyEnvironmentId) throws Exception {
+    protected List<ModuleExportInfo> generateSurveyModules(ExportOptions exportOptions, UUID studyEnvironmentId) throws Exception {
```

Comment:
Does this throw a particular exception (`Exception` is pretty broad)? Any reason that cannot be caught and converted in this method?